virttest/qemu_devices/qcontainer.py

```diff
@@ -1678,6 +1678,7 @@ def images_define_by_variables(self, name, filename, pci_bus, index=None,
                                    num_queues=None, bus_extra_params=None,
                                    force_fmt=None, image_encryption=None,
                                    image_access=None, external_data_file=None,
+                                   image_copy_on_read=None,
```

Comment:
move this param to the end (after image_discard) in case this function is called somewhere other than qcontainer.py

Reply:
Have updated, thanks zhenchao.